### DISCUSSION TITLE
重学 Vim - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#40] [Announcements]
# 重学 Vim - forecho's Blog

引言
2013 年的时候我才开始接触 Vim，简单学了一下，也没用起来，后来只记得普通模式以及怎么进入插入模式，然后就是保存退出这几个命令了。
直到最近买了一个 HHKB 键盘以及发现使用 Vim 编码效率要高很多，所以我决定重拾 Vim，为了加强自己的记忆，于是我就整理了这篇文章。

https://blog.forecho.com/re-study-vim.html

**Comment by daylenjeez** [comment]
哭死，这也太难记了吧😭

**Reply by forecho** [reply]
常用的记住就行，多用，形成肌肉记忆
